docs/ja/authenticators.rst

```diff
@@ -192,173 +170,154 @@ configuring your app as follows::
         $this->set(compact('keys'));
         $this->viewBuilder()->setOption('serialize', 'keys');
     }
-    
-Refer to https://tools.ietf.org/html/rfc7517 or https://auth0.com/docs/tokens/concepts/jwks for
-more information about JWKS.
 
-HttpBasic
+JWKSの詳細情報は https://tools.ietf.org/html/rfc7517
+または https://auth0.com/docs/tokens/concepts/jwks を参照してください。
+
+Http基本
 =========
 
-See https://en.wikipedia.org/wiki/Basic_access_authentication
+https://en.wikipedia.org/wiki/Basic_access_authentication を確認してください。
 
-Configuration options:
+設定オプション:
 
--  **realm**: Default is ``$_SERVER['SERVER_NAME']`` override it as
-   needed.
+-  **realm**: デフォルトは ``$_SERVER['SERVER_NAME']``  で、必要に応じて上書きしてください。
 
-HttpDigest
-==========
+Httpダイジェスト
+=================
 
-See https://en.wikipedia.org/wiki/Digest_access_authentication
+https://en.wikipedia.org/wiki/Digest_access_authentication を確認してください。
 
-Configuration options:
+設定オプション:
 
--  **realm**: Default is ``null``
--  **qop**: Default is ``auth``
--  **nonce**: Default is ``uniqid(''),``
--  **opaque**: Default is ``null``
+-  **realm**: デフォルトは ``null`` です。
+-  **qop**: デフォルトは ``auth`` です。
+-  **nonce**: デフォルトは ``uniqid(''),`` です。
+-  **opaque**: デフォルトは ``null`` です。
 
-Cookie Authenticator aka "Remember Me"
+クッキー認証機能 別名 "リメンバーミー"
 ======================================
 
-The Cookie Authenticator allows you to implement the “remember me”
-feature for your login forms.
+クッキー認証機能を使用すると、ログインフォームに "remember me "機能を実装することができます。
 
-Just make sure your login form has a field that matches the field name
-that is configured in this authenticator.
+ログインフォームに、この認証機能で設定されているフィールド名と一致するフィールドがあることを確認してください。
 
-To encrypt and decrypt your cookie make sure you added the
-EncryptedCookieMiddleware to your app *before* the
-AuthenticationMiddleware.
+クッキーを暗号化して複合化するには、AuthenticationMiddlewareの **前に**
+EncryptedCookieMiddlewareをアプリに追加したことを確認してください。
 
-Configuration options:
+設定オプション:
 
--  **rememberMeField**: Default is ``remember_me``
--  **cookie**: Array of cookie options:
+-  **rememberMeField**: デフォルトは ``remember_me`` です。
+-  **cookie**: クッキーオプションの配列:
 
-   -  **name**: Cookie name, default is ``CookieAuth``
-   -  **expire**: Expiration, default is ``null``
-   -  **path**: Path, default is ``/``
-   -  **domain**: Domain, default is an empty string \`\`
-   -  **secure**: Bool, default is ``false``
-   -  **httpOnly**: Bool, default is ``false``
-   -  **value**: Value, default is an empty string \`\`
+-  **name**: クッキー名, デフォルトは ``CookieAuth``
```

Comment:
This item needs to be indented further.

Reply:
Sorry. Sorry for lack of confirmation. ....
Please check under the request to push the one you just fixed.
Thank you for finding the mistake.
Sorry for the late reply.